src/UI/Badge.elm

```diff
@@ -143,13 +190,10 @@ withBrightness brightness (Badge prop opt) =
 The result of this function is a ready-to-insert Elm UI's Element.
 -}
 renderElement : RenderConfig -> Badge -> Element msg
-renderElement cfg (Badge { content, tone } { brightness }) =
+renderElement cfg (Badge { content, tone, brightness }) =
```

Comment:
Also 2 small things there is not exactly related to this pr, but we can fix it here :D 

1. Padding

![Screenshot 2020-07-29 at 12 19 01](https://user-images.githubusercontent.com/1559013/88788541-c0035e80-d195-11ea-83ba-e3a125563a5c.png)

2. Border radius

We are using 8, but the badges are using 10